tests/resources/test_data_source_dir/test_data_source_data.py

```diff
@@ -85,6 +84,54 @@ def execute(self, sql: str):
 
 
 oracledb_real_data = [
+    Row(
+        ID=1,
+        NUMBER_COL=Decimal("2720.24"),
+        BINARY_FLOAT_COL=2170.613525390625,
+        BINARY_DOUBLE_COL=263.5460877388568,
+        VARCHAR2_COL="zKtAGe0Jrv83Um9qDyKHDRISQ2JyeI7EVAoTigJsCE54x1pWCS",
+        CHAR_COL="Oz8xoFidXB",
```

Comment:
Are we keeping/adding-back some mock data? The PR description says we will use real data and no mock data

Reply:
We are removing mock data of oracledb in the PR.
The data you refer to is the expected result of oracledb, it is a bit different from what you directly get from oracledb, so I just put it here